pkg/ocm/client.go

```diff
@@ -0,0 +1,52 @@
+package ocm
+
+import (
+	"context"
+	"errors"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+)
+
+type Client struct {
+	logger sdk.Logger
+}
+
+func NewClient() (*Client, error) {
+	logger, err := sdk.NewGoLoggerBuilder().
+		Debug(true).
+		Build()
+	if err != nil {
+		return nil, err
+	}
+
+	return &Client{logger: logger}, nil
+}
+
+func (c *Client) GetOrganization(ctx context.Context, authToken string, orgID string) (string, error) {
+	// Create the connection, and remember to close it:
+	connection, err := sdk.NewConnectionBuilder().
```

Comment:
I just wonder how this will work on stage. I think we need to have an option to switch the envs.